packages/remark-lint-blockquote-indentation/readme.md

```diff
@@ -157,16 +157,16 @@ Due to this, it’s recommended to configure this rule with `2`.
 
 ##### `ok.md`
 
-When configured with `2`.
+When configured with `4`.
```

Comment:
I also wrapped `{config}` in an object. It's the `tests` key, which changes the order of some examples, because the `tests` JS object isn't order preserving: https://github.com/remarkjs/remark-lint/pull/279#discussion_r810683762

Before some examples were in numeric order, now they're in the order defined in the docblocks. Before the key was `'4'`, now it's `'{"config":4}'`.

This is also a cosmetic change --- it extracts the change in example order from https://github.com/remarkjs/remark-lint/pull/279#pullrequestreview-901858269. That PR changes the `tests` key from `{config}` -> `{settings, config}`.